stable_baselines3/common/logger.py

```diff
@@ -99,6 +110,9 @@ def writekvs(self, kvs: Dict) -> None:
 
     @classmethod
     def _truncate(cls, string: str) -> str:
+        # Remove tensorboard tag from std outputs
+        if string.find('/') > 0:
```

Comment:
why? it more readable with the tag in std out, no?

Reply:
Some keys would get quite long and being truncated because of the tags. Maybe we can output the tag as a parent and the keys under as children?

Reply:
nevermind, I've enlarged the truncate threshold a bit and it looks nice as it is as well

Reply:
>Some keys would get quite long and being truncated because of the tags. Maybe we can output the tag as a parent and the keys under as children?

this is a good idea

Reply:
Ah, okay